### PR TITLE
New version: whitewaterfoundry.fedora-remix-for-wsl version 35.12.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37534)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: whitewaterfoundry.fedora-remix-for-wsl
+PackageVersion: 35.12.3
```
Comment (ItzLevvie) [inline manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 35.12.3.0
```

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: whitewaterfoundry.fedora-remix-for-wsl
+PackageVersion: 35.12.3
```
Comment (ItzLevvie) [inline manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.installer.yaml:5]:
```suggestion
PackageVersion: 35.12.3.0
```

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: whitewaterfoundry.fedora-remix-for-wsl
+PackageVersion: 35.12.3
```
Comment (ItzLevvie) [inline manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.yaml:5]:
```suggestion
PackageVersion: 35.12.3.0
```

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.installer.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.installer.yaml:5]: image

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.locale.en-US.yaml:5]: image

File: manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/w/whitewaterfoundry/fedora-remix-for-wsl/35.12.3/whitewaterfoundry.fedora-remix-for-wsl.yaml:5]: image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**